packages/website/docs/Internal/Grammar.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Grammar
+urlcolor: blue
```

Comment:
why is blue specified in this file only?

Reply:
I think it was an artefact from frontmatter I copied over from the `invariants` document, which was an artefact of `pandoc` (because Nuño initially wrote that file for `pandoc` rendering)

Reply:
removed, tho.